examples/cli/src/main/java/io/unitycatalog/cli/MetastoreCli.java

```diff
@@ -0,0 +1,37 @@
+package io.unitycatalog.cli;
```

Comment:
can you make a separate PR for the doc update? lets put [0.3] in the PR title so that we can find and merge it when we are closer to 0.3 release

Reply:
will do!